build.psm1

```diff
@@ -981,11 +976,12 @@ function Publish-PSTestTools {
 function Start-PSPester {
     [CmdletBinding(DefaultParameterSetName='default')]
     param(
+        [Parameter(Position=0)]
+        [string[]]$Path = @("$PSScriptRoot/test/common","$PSScriptRoot/test/powershell"),
```

Comment:
In your PR description, you wrote:
> Make `-Path` parameter on `Start-PSPester` the 0th parameter to so that `Restore-PSPester $Path` works without needing to specify -Path as a named parameter.

I'm confused about the `Restore-PSPester $Path` part. You are not calling `Restore-PSPester` with `$Path` in your change.

Reply:
This change was for easier interactive usage only

Reply:
You mentioned about `Restore-PSPester $Path` in your description, but I don't see how making `-Path` positional affect your use of `Restore-PSPester`.

Do you mean `without needing to specify -Path for Start-PSPester` ?

Reply:
Sorry, my bad. I meant to write `Start-PSPester $Path`, you are right

Reply:
Can you please update your PR description? Thanks!

Reply:
Ok, done. Sorry for that mistake.